index.html

```diff
@@ -0,0 +1,334 @@
+					</div>
+					<div class="btn-project">
+						<a href="">See Project</a>
+					</div>
+				</div>
+			</div>
+			<div class="details-card">
+				<div class="project-image"><img src="./images/project2.svg" alt="projet1img"></div>
+				<div class="card-info">
+					<div class="project-description">
+						<div class="project-details">
+							<h1 class="project-title">Multi-Post Stories</h1>
+							<div class="client-info">
+								<p class="canopy">CANOPY</p>
+								<ul class="role">
+									<li>Back End Dev</li>
+									<li>2015</li>
+								</ul>
+							</div>
+
+						</div>
+						<div class="project-info">
+							<p>
+								A daily selection of privately personalized reads;
+								no accounts or sign-ups required.
+							</p>
+						</div>
+						<div class="project-tag">
+							<ul class="lang">
+								<li>html</li>
+								<li>css</li>
+								<li>javascript</li>
+							</ul>
+						</div>
+					</div>
+					<div class="btn-project">
+						<a href="">See Project</a>
+					</div>
+
+				</div>
+			</div>
+			<div class="details-card">
+				<div class="project-image"><img src="./images/project3.svg" alt="project1img"></div>
+				<div class="card-info">
+					<div class="project-description">
+						<div class="project-details">
+							<h1 class="project-title">Tonic</h1>
+							<div class="client-info">
+								<p class="canopy">CANOPY</p>
+								<ul class="role">
+									<li>Back End Dev</li>
+									<li>2015</li>
+								</ul>
+							</div>
+						</div>
+						<div class="project-info">
+							<p>
+								A daily selection of privately personalized reads;
+								no accounts or sign-ups required.
+							</p>
+						</div>
+						<div class="project-tag">
+							<ul class="lang">
+								<li>html</li>
+								<li>css</li>
+								<li>javascript</li>
+							</ul>
+						</div>
+					</div>
+					<div class="btn-project">
+						<a href="">See Project</a>
+					</div>
+
+				</div>
+			</div>
+			<div class="details-card">
+				<div class="project-image"><img src="./images/project4.svg" alt="projet1img"></div>
+				<div class="card-info">
+					<div class="project-description">
+						<div class="project-details">
+							<h1 class="project-title">Multi-Post Stories</h1>
+							<div class="client-info">
+								<p class="canopy">CANOPY</p>
+								<ul class="role">
+									<li>Back End Dev</li>
+									<li>2015</li>
+								</ul>
+							</div>
+
+						</div>
+						<div class="project-info">
+							<p>
+								A daily selection of privately personalized reads;
+								no accounts or sign-ups required.
+							</p>
+						</div>
+						<div class="project-tag">
+							<ul class="lang">
+								<li>html</li>
+								<li>css</li>
+								<li>javascript</li>
+							</ul>
+						</div>
+					</div>
+					<div class="btn-project">
+						<a href="">See Project</a>
+					</div>
+
+				</div>
+			</div>
+		</main>
+
+		<!-- About Me section -->
+
+		<main id="section-about">
```

Comment:
- kindly, it is better to use the `section` tag here because the `<main>` element should be unique to the document. and must not be more than one, check this for [more info](https://www.w3schools.com/tags/tag_main.asp) :+1: